src/extension/background-script/actions/lnurl/pay.ts

```diff
@@ -0,0 +1,20 @@
+import utils from "../../../../common/lib/utils";
+
+import type { Message, LNURLDetails } from "../../../../types";
+
+async function payWithPrompt(message: Message, lnurlDetails: LNURLDetails) {
+  // for lnurl pay requests we always open a prompt
+  // Question: does it help to support an allowance here?
```

Comment:
For who is the question?

Reply:
not sure for whom :) for me while working on it maybe :D